### PR TITLE
Initial draft of herbarium ADR

### PR DESCRIPTION
**Pull request by ZavenArra**

### CODE REVIEW

File: 0011-herbarium-system.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Decision Drivers <!-- optional -->
+
+* {driver 1, e.g., a force, facing concern, …}
+* {driver 2, e.g., a force, facing concern, …}
+* … <!-- numbers of drivers can vary -->
+
+## Considered Options
+
+1. Implement an herbarium microservice to provide organization specific common names, while aggregating globally relevant species information.
+2. Alternative option?
```
Comment (camwebb) [inline 0011-herbarium-system.md:27]: The hidden complexity of is that if we allow each organization to see a short list of local names for their organization, the mapping between each local name and a scientific name and/or taxon code needs to be examine carefully. Who will do this? This multiplies the work of keeping species information accurate many times. For example: an org (orgX) says "we plant ironwood", and either we at GS or the org people say "we think ironwood is taxon Z", and we add "ironwood"@orgX to the names for taxon Z without checking, and then we server "ironwood" as a name for orgX in the admin panel. Unless `"ironwood"@orgX = taxon Z` has been checked, we may just be adding garbage to the DB. To check this, someone needs to look at several seedlings of "ironwood" owned by orgX.  To do this checking for every species x org combination is a huge amount of work.  The current systems places the onus on the org, to make their own matching of their local name to a taxon code.
```suggestion
2. Offer a set of standardized taxa codes and names, filtered by the planting list provided by an organization.
```

**Review by camwebb — COMMENTED** [review]
I think there is a need to step back and examine the species information system in general, before or at least alongside this particular ADR for common names.  I'll begin a new ADR (please allow a few days to get my thoughts in order).

File: 0011-herbarium-system.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Design of the herbarium system
```
[the remark this answers is not in the document]
Comment (ZavenArra) [reply 0011-herbarium-system.md:1]: Seems like this _will_ be the herbarium service, with the determinations service being the common names service

File: 0011-herbarium-system.md (hunk 1 above)
Comment (ZavenArra) [reply 0011-herbarium-system.md:27]: I think 0012 is the alternative option here, right?

**Comment by ZavenArra** [comment]
ADR0012 suggests a new service, which I am calling 'determinations service', which deals with mapping captures to common names to taxa.  This leaves the species specific information itself to reside in the 'herbarium' service, which I suggest becomes what this ADR tracks.

File: 0011-herbarium-system.md (hunk 1 above)
Comment (camwebb) [reply 0011-herbarium-system.md:27]: Yes

File: 0011-herbarium-system.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (camwebb) [reply 0011-herbarium-system.md:1]: Agree. ADR 12 is now the determination service. And ADR 11 is the herbarium service.

File: 0011-herbarium-system.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-06-07
+
+Technical Story: 
+* https://github.com/Greenstand/treetracker-herbarium-api/issues/1 
+* https://www.figma.com/file/RrG8fahXei5SDY6AYjYYnn/Digital-Herbarium---Species%2FCommon-Names?node-id=0%3A1
+
+## Context and Problem Statement
+
+Tree planting and tracking organizations use various, localy relevant common names to identify the species of tree they are tracking.  This leads to inconsistencies when identifying which species was actually tracked.  Treetracker requires an herbarium service to connect these common names to the globally recognized scientific name.  The herbarium service can also also server as repository of identifying information and socially or ecologically relevant parameters for tree species tracked by organizations.
```
[the remark this answers is not in the document]
Comment (camwebb) [reply 0011-herbarium-system.md:15]: > do you think the herbarium microservice should be separate from the determinations microservice

Yes.

File: 0011-herbarium-system.md (hunk 1 above)
Comment (ZavenArra) [reply 0011-herbarium-system.md:27]: Great.

**Comment by camwebb** [comment]
Okay, so he have two services: determinations (ADR12) and plant species data (aka "herbarium" ADR11). At the moment, simply dumping info from the [XML file](https://github.com/Greenstand/Tree_Species/blob/master/tree_species.xml) I edit into one or more DB tables, and then serving this info via a microservice, seems the way to go. 

So far no one has really used that XML file and the schema is still flexible. If it gets imported into herbarium DB tables, I'll have to freeze the schema.

**Comment by camwebb** [comment]
@ZavenArra do you want to merge this to have it in the record, and then further edit it via PRs. Or work more on the text of the first PR?

**Comment by ZavenArra** [comment]
lemme ponder that, it does get ungainly with the open PRs and changes and comments to the PR but the process recommended is not to merge until the PR is accepted.  The process needs to be improved upon.

**Comment by sebastiangaertner** [comment]
@ZavenArra where are we on this? We have a strategic partner who is interested in us developing a global API based herbarium and we are in a good position to do so. We could focus our grant searches on that as well.

**Comment by sebastiangaertner** [comment]
@camwebb we are trying to get Northeastern University to get involved building this as a free(ish) standing API (still hosted by Greenstand since we have the capacity and will be main users). How can we advance this?

**Comment by camwebb** [comment]
> How can we advance this?

Not sure what needs advancing. Who is offering from Northeastern? What are they offering?

**Comment by dadiorchen** [comment]
I think to make it a open API there are two direction:
* A open online service and Greenstand as one of the client.
* An opensource software that Greenstand will host it on its own cloud and any other organization who want to use it can install and host it by themselves.

**Comment by camwebb** [comment]
Thanks @dadiorchen 

But @sebastiangaertner, to answer this question, I still need to know which is the team at Northeastern... are they botanists? Are they offering to do the species detective work? Or just to create an infrastructure? What would their API offer? Would it be based on GS images? Thanks

**Comment by camwebb** [comment]
I realized this PR is still open when I submitted the new PR for 0012-species-data. Closing this one.

**Pull request closed without merging**